Dshop/apps/products_catalogue/tests/test_api_products.py

```diff
@@ -42,6 +42,18 @@ def authenticated_api_client(api_client, user_instance_token):
     return api_client
 
 
+def assert_active_object(data):
+    fields_values = {
+        "name": "main product",
+        "price": "11.00",
+        "short_description": "short desc",
+        "full_description": "full_description",
+    }
+    for key, value in data.items():
+        if key in fields_values:
```

Comment:
please remove this if. 
The goal of the test is to FAIL if something is missing, this is so called "silent failure" - test passes, you expect all good but no, field is not there. 
Example: data = {}
Test will pass.